src/charm.py

```diff
@@ -60,18 +58,10 @@ def __init__(self, *args):
             name="tempo",
             pebble_layer=self.generate_worker_layer,
             endpoints={"cluster": "tempo-cluster"},  # type: ignore
+            readiness_check_endpoint="http://localhost:3200/ready",
```

Comment:
Might be worth checking if it will still work with TLS on. I wonder if http endpoint might start to refuse traffic then.

Reply:
uh, good thinking. will try that out

Reply:
![image](https://github.com/user-attachments/assets/9dac4a7a-93de-48df-9476-32bfaa4dfd57)

ouch, you were right

Reply:
fix depends on https://github.com/canonical/cos-lib/pull/59